src/telcameras_v2/data_conversions.py

```diff
@@ -0,0 +1,36 @@
+def data_to_observation(data):
```

Comment:
Deze method heeft een stukje uitleg nodig over wat de input is en wat de output is